src/main/java/teammates/ui/pagedata/FeedbackSubmissionEditPageData.java

```diff
@@ -233,24 +243,45 @@ private FeedbackSubmissionEditQuestion createQuestion(FeedbackQuestionAttributes
         List<FeedbackSubmissionEditResponse> responses = new ArrayList<>();
 
         List<FeedbackResponseAttributes> existingResponses = bundle.questionResponseBundle.get(questionAttributes);
+        List<String> responseSubmittedRecipient = new ArrayList<String>();
         int responseIndx = 0;
-
+        // to generate options when responseIndx < numOfResponseBoxes.
+        int i = 0;
```

Comment:
Declare this variable closer to where it will be used; the following `for` loop doesn't use `i` at all.